controller/workitem_blackbox_test.go

```diff
@@ -2416,6 +2416,90 @@ func (s *WorkItem2Suite) TestDefaultSpaceAndIterationRelations() {
 	assert.Equal(s.T(), rootIteration.ID.String(), *wi.Data.Relationships.Iteration.Data.ID)
 }
 
+// Following test verifies that UPDATE on WI by setting AREA & Iteration
+// works as expected and do not alter previously set values
+func (s *WorkItem2Suite) TestWI2UpdateWithAreaIterationSuccessively() {
```

Comment:
Missing test case of setting Area or Iteration 'back to default' on Data = nil ?